frontera/logger/formatters/color.py

```diff
@@ -10,7 +10,7 @@
 class ColorFormatter(ColoredFormatter):
 
     def __init__(self, format, log_colors, log_color_field, datefmt=None, reset=True, style='%'):
-        super(ColorFormatter, self).__init__(format=format, datefmt=datefmt, log_colors=log_colors,
+        super(ColorFormatter, self).__init__(fmt=format, datefmt=datefmt, log_colors=log_colors,
```

Comment:
It looks like guys changed the method signature. See https://github.com/borntyping/python-colorlog/blob/v2.6.0/colorlog/colorlog.py#L67
What if we use positional arguments for `format` instead of key-valued? Will it bring us more resistance to signature changes?